msgraph-connector-demo/src/msgraph/wrappers/MsGraphRestWrapper.java

```diff
@@ -0,0 +1,41 @@
+package msgraph.wrappers;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import com.microsoft.graph.CollectionOfMessage;
+import com.microsoft.graph.MicrosoftGraphMessage;
+
+import ch.ivyteam.ivy.environment.Ivy;
+
+public class MsGraphRestWrapper {
```

Comment:
Very nice encapsulation of "just" the rest part of the problem. 
As a suggestion, you may leave out the "Wrapper" postfix. It doensn't really add more meaning to the class.